README.md

```diff
@@ -3,7 +3,7 @@ Some tools and scripts for a UPS test suite
 
 ## Mocked Data
 
-A tool to produce mock data (push applications, variants and tokens) can be found into the [ups-test-suite/mock-data-loader](ups-test-suite/mock-data-loader) folder.
+A tool to produce mock data (push applications, variants and tokens) can be found into the [mock-data-loader](mock-data-loader) folder.
```

Comment:
I would include this section below _Automated Tools_, with the other tools, and change _Automated Tools_ into plain _Tools_.